README.md

```diff
@@ -11,47 +11,48 @@ A sample MLOps system of LeapMind Blueoil x AWS Components (AWS IoT Greengrass,
 ### Requirements
 You need following packages to setup components.
-* AWS CLI
+#### AWS CLI
 
-Please set up the AWS Command Line Interface (AWS CLI), check [official refferences](https://docs.aws.amazon.com/polly/latest/dg/setup-aws-cli.html).
-* Ansible
+Please set up the AWS Command Line Interface (AWS CLI), check [official User Guide](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html). You can choose the region where Amazon SageMaker and AWS IoT Greengrass are supported (see [Region Table](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/)). 
+#### Ansible
 
 You can install with `pip` command.
 ```shell
 $ pip3 install -r requirements.txt
 ```
 
-### Prepare your S3 bucket and Roles for deployment
-Decide your S3 bucket name. Bucket names must be unique across all existing bucket names in Amazon S3. Please see [Bucket Restrictions and Limitations](https://docs.aws.amazon.com/AmazonS3/latest/dev/BucketRestrictions.html).
+### Prepare your Amazon S3 bucket and Roles for deployment
+Decide your Amazon S3 bucket name. Bucket names must be unique across all existing bucket names in Amazon S3. Please see [Bucket Restrictions and Limitations](https://docs.aws.amazon.com/AmazonS3/latest/dev/BucketRestrictions.html).
 ```shell
 $ export BLUEGRASS_S3_BUCKET_NAME=[your S3 bucket name]
 $ aws cloudformation create-stack --stack-name BluegrassS3 --capabilities CAPABILITY_NAMED_IAM --template-body file://$(pwd)/deploy/s3.yaml --parameters ParameterKey="S3BucketName",ParameterValue="${BLUEGRASS_S3_BUCKET_NAME}"
 ```
 Please make sure your bucket is created. Check status from your [cloudformation console](https://console.aws.amazon.com/cloudformation/home).
 
-### Blueoil x SageMaker
-#### Create SageMaker Notebook for training with Blueoil
+### Blueoil x Amazon SageMaker
+#### Create an Amazon SageMaker Notebook instance for training with Blueoil
 ```shell
 $ aws cloudformation create-stack --stack-name BlueoilSagemaker --template-body file://$(pwd)/deploy/sagemaker.yaml
 ```
 
-#### Run training on SageMaker Notebook
-See [blueoil_sagemaker/README.md](blueoil_sagemaker/README.md).
+#### Run training on Amazon SageMaker 
+Open Jupyter/JupyterLab on the [Amazon SageMaker console](https://console.aws.amazon.com/sagemaker/). Find and run the notebook in the `bluegrass/blueoil_sagemaker` directory, e.g., [`blueoil_cifar10_example.ipynb`](./blueoil_sagemaker/blueoil_cifar10_example.ipynb) or [`blueoil_openimages_example.ipynb`](./blueoil_sagemaker/blueoil_openimages_example.ipynb) .  
 
-### Blueoil x Greengrass
-#### Create certificate files for Greengrass
+### Blueoil x AWS IoT Greengrass
+#### Create certificate files for AWS IoT Greengrass
 Run following script.
 ```shell
 $ deploy/create_cert.sh
 ...
 Certificate files are created in certs/xxxxxx...xxxxxx
-certificateArn is "arn:aws:iot:xxxxxx:xxxxxx:cert/xxxxxx...xxxxxx"
+certificateArn is "arn:aws:iot:ap-northeast-1:123456789012:cert/xxxxxx...xxxxxx"
```

Comment:
NIT: Do we need to change this line? I think the value of `ap-northeast-1:123456789012` depends by the user's environment.